src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -1270,13 +1270,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
     }
 
     /**
-     * Post /courses/:courseId/instructors/:instructorLogin : Add the given user to the instructors of the course so that the student can access the course administration
+     * POST courses/:courseId/instructors/:instructorLogin : Add the given user to the instructors of the course so that the student can access the course administration
```

Comment:
```suggestion
     * Post /courses/:courseId/instructors/:instructorLogin : Add the given user to the instructors of the course so that the student can access the course administration
```